source/connect/ssh-connection.txt

```diff
@@ -11,3 +11,157 @@ Proxy / SSH Tunnel Connection Tab
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+The :guilabel:`Proxy / SSH Tunnel` tab allows you to connect your deployment using 
+a Proxy method or SSH Tunnel. 
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`Proxy / SSH Tunnel` tab.
+
+      You have the following connection options:
+
+      .. list-table::
+         :header-rows: 1 
+         :widths: 50 50 
+      
+         * - Option
+           - Description 
+
+         * - SSH with Password 
+           - Connects a deployment using a SSH Tunnel and has the option to secure 
+             the connection with a password. For information on SSH connections and 
+             the fields found under this connection option, see :ref:`<ssh-fields>`.
+
+         * - SSH with Identity File 
+           - Connects a deployment using a SSH Tunnel and uses an Identity File 
+             to secure the connection. For information on SSH connections and 
+             the fields found under this connection option, see :ref:`<ssh-fields>`.
+
+         * - Socks5
+           - Connects a deployment using a seperate proxy server. For more 
+             information on the fields found under this connection option, see 
+             :ref:`<socks5>`. 
+
+      .. _ssh-fields:
+ 
+      SSH Connection
+      ~~~~~~~~~~~~~~
+
+      The following fields are used when connecting a deployment using 
+      :guilabel:`SSH with Password` or :guilabel:`SSH with Identity File`:
+
+      .. list-table::
+         :header-rows: 1 
+         :widths: 50 50
+      
+         * - Field
+           - Description
+
+         * - SSH Hostname
+           - Enter the bastion (jumpbox) hostname. This is the unique identifier 
+             (Fully Qualified Domain Name, or FQDN) for the machine to be accessed. 
+
+         * - SSH Port
+           - Provide the port used for the SSH connection. This defaults to 22, 
```

Comment:
I noticed the descriptions sometimes include "The" and some omit. I would standardize on one. Please make the change throughout the update. Cheers.

Reply:
Done.